packages/dd-trace/src/appsec/remote_config/manager.js

```diff
@@ -299,4 +319,22 @@ function parseConfigPath (configPath) {
   }
 }
 
+function supportsAckCallback (handler) {
+  if (kSupportsAckCallback in handler) return handler[kSupportsAckCallback]
+
+  const numOfArgs = handler.length
+  let result = false
+
+  if (numOfArgs >= 4) {
+    result = true
+  } else if (numOfArgs !== 0) {
+    const source = handler.toString()
+    result = source.slice(0, source.indexOf(')')).includes('...')
+  }
```

Comment:
oh wow that's cursed, whould anyone even want to use spread operator anyway ? like do we really need that special case ?

Reply:
We don't currently use it or have a need for it, but if we don't have this check, the code is going to behave unexpected if someone at some point decides to use it. So writing it this way is safer